defaults/main.yml

```diff
@@ -8,6 +8,18 @@ sensu_config_base_path: /etc/sensu
 sensu_config_path: "{{ sensu_config_base_path }}/conf.d"
 sensu_config_file_path: "{{ sensu_config_base_path }}/config.json"
 
+sensu_apt_package: sensu
+sensu_apt_key_url: https://sensu.global.ssl.fastly.net/apt/pubkey.gpg
+sensu_deb_url: deb https://sensu.global.ssl.fastly.net/apt {{ ansible_distribution_release }} main
```

Comment:
No double quote when using variable?

Reply:
It is variable. I will edit use double quote.